portal/src/components/FilterCategories/FilterCategories.component.js

```diff
@@ -7,24 +6,26 @@ export default {
   name: 'filter-categories',
   components: { DatesRange },
   data() {
-    const publisherdate = 'lom.lifecycle.contribute.publisherdate'
+    const publisherDate = 'lom.lifecycle.contribute.publisherdate'
     return {
-      publisherdate,
-      visible_items: 20,
+      publisherDate,
       data: {
         start_date: null,
         end_date: null
       }
     }
   },
+  props: ['filterCategories', 'materials', 'selectedFilters'],
   methods: {
     generateSearchMaterialsQuery,
     hasVisibleChildren(category) {
       if (!category.children.length) {
         return false
       }
       return _.some(category.children, child => {
-        return !child.is_hidden
+        if (child.count > 0) {
```

Comment:
`return child.count > 0 && !child.is_hidden`

Reply:
Dit is problematisch. Een child met een count van 0 is niet altijd hidden. Misschien disabled?

Reply:
wil je dan alle children laten zien? ook als ze geen resultaten hebben? wat ik al eerder zei: het is me niet zo duidelijk hoe het moet werken.